crypto/secp256k1/secp256k1.go

```diff
@@ -37,25 +37,24 @@ func init() {
 		PrivKeyAminoName, nil)
 }
 
-//-------------------------------------
-
 var _ crypto.PrivKey = PrivKeySecp256k1{}
 
-// PrivKeySecp256k1 implements PrivKey.
-type PrivKeySecp256k1 [32]byte
+// PrivKey implements PrivKey.
+type PrivKeySecp256k1 []byte
```

Comment:
what is the motivation to change from array to slice? Is it possible to have more than 32 or less than 32 bytes now?